frontend/src/metabase/core/utils/errors/messages.ts

```diff
@@ -0,0 +1,24 @@
+import type { GenericErrorResponse } from "./types";
+
+export function getResponseErrorMessage(
+  error: GenericErrorResponse,
```

Comment:
Should this function accept `unknown` instead? Every call place needs to cast it now.